wire-tests-swift/src/main/swift/module_two/SwiftModuleTwoMessage.swift

```diff
@@ -0,0 +1,104 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source: squareup.protos.kotlin.swift_module_two.SwiftModuleTwoMessage in swift_module_two.proto
+import Foundation
+import Wire
+import module_one
+
+public struct SwiftModuleTwoMessage {
+
+    public var name: String?
+    public var module_one_names: [SwiftModuleOneMessage] = []
+    public var unknownFields: Foundation.Data = .init()
+
+    public init(configure: (inout Self) -> Swift.Void = { _ in }) {
+        configure(&self)
+    }
+
+}
+
+#if WIRE_INCLUDE_MEMBERWISE_INITIALIZER
+extension SwiftModuleTwoMessage {
+
+    @_disfavoredOverload
+    @available(*, deprecated)
+    public init(name: Swift.String? = nil, module_one_names: [module_one.SwiftModuleOneMessage] = []) {
+        self.name = name
+        self.module_one_names = module_one_names
+    }
+
+}
+#endif
+
+#if !WIRE_REMOVE_EQUATABLE
+extension SwiftModuleTwoMessage : Equatable {
+}
+#endif
+
+#if !WIRE_REMOVE_HASHABLE
+extension SwiftModuleTwoMessage : Hashable {
+}
+#endif
+
+#if swift(>=5.5)
+extension SwiftModuleTwoMessage : Sendable {
+}
+#endif
+
+extension SwiftModuleTwoMessage : ProtoMessage {
+
+    public static func protoMessageTypeURL() -> Swift.String {
+        return "type.googleapis.com/squareup.protos.kotlin.swift_module_two.SwiftModuleTwoMessage"
+    }
+
+}
+
+extension SwiftModuleTwoMessage : Proto2Codable {
+
+    public init(from protoReader: Wire.ProtoReader) throws {
+        var name: Swift.String? = nil
+        var module_one_names: [module_one.SwiftModuleOneMessage] = []
```

Comment:
You should add a second .proto which has a map of cross module types.

Reply:
yeap agree

Reply:
done and a test case.